lib/fennec/stun/auth.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Fennec.STUN.Auth do
+  @moduledoc """
+  This module implements authentication and authorization of the STUN protocol.
+  """
+
+  alias Jerboa.Format.Body.Attribute
+  alias Jerboa.Format.Body.Attribute.{Username, Nonce, Realm}
+  alias Jerboa.Params
+
+  def get_secret do
+    Application.get_env(:fennec, :secret)
+  end
+
+  def authorize(params, _turn_state) do
+    with u = %Username{} <- Params.get_attr(params, Username),
+         r = %Realm{}    <- Params.get_attr(params, Realm) do
+      # Right now, any authenticated user will do
+      {:ok, %Params{params | attributes: params.attributes -- [u, r]}}
```

Comment:
I see. I'm afraid that if someone comes back to Fennec and e.g. would like to log params going through `Evaluator`, and sees no Username and Realm, it might be a little confusing.